avm/res/data-factory/factory/README.md

```diff
@@ -165,7 +165,7 @@ module factory 'br/public:avm/res/data-factory/factory:<version>' = {
         }
```

Comment:
This is just a blocking comment to ensure this PR is only merged after https://github.com/Azure/bicep-registry-modules/pull/3149.